src/components/CardList/CardList.tsx

```diff
@@ -1,9 +1,15 @@
 import styles from "./cardlist.module.css";
 import Card from "components/Card/Card";
 import { NoResults } from "pages";
+import { GetLinkResponse } from "types/apis";
+import { UseModal } from "utils/hooks/useModal";
 
-function CardList({ items, ...rest }) {
-  const handleClick = (url) => {
+interface Props extends Partial<UseModal> {
+  items: GetLinkResponse[] | null;
```

Comment:
링크를 가져오는 응답값 타입의 배열이라는게 여기서도 어색하네요~
이렇게 표현되면 좋을 것 같아요!
```suggestion
  items: LinkItem[] | null;
```